src/Distribution/Nixpkgs/Haskell/Stack/PrettyPrinting.hs

```diff
@@ -74,10 +75,15 @@ overrideHaskellPackages oc packages = vcat
     , "};"
     , ""
     ]
-  , "in callPackage <nixpkgs/pkgs/development/haskell-modules> {"
+  , "in callPackage (nixpkgsPath + \"/pkgs/development/haskell-modules\") {"
   , nest 2 $ vcat
     [ attr "ghc" ("pkgs.haskell.compiler." <> toNixGhcVersion (oc ^. ocGhc))
     , attr "compilerConfig" "self: extends pkgOverrides (extends stackageConfig (stackagePackages self))"
+    , "haskellLib = import (nixpkgsPath + \"/pkgs/development/haskell-modules/lib.nix\") {"
```

Comment:
Try to use here `callPackage` instead of `import`. It should provide `pkgs` and `lib` attributes. Then you won't need to inherit them explicitly.